src/celeritas/ext/detail/GenericSolid.hh

```diff
@@ -209,12 +219,12 @@ class GenericSolid : public vecgeom::VUnplacedVolume
     }
     DevicePtr<CudaUnplacedVolume> CopyToGpu() const
     {
-        return {};
+        CELER_NOT_IMPLEMENTED("GenericSolid with CUDA");
```

Comment:
So, will any generic solid trigger an exception when CELERITAS_USE_CUDA=ON (even before the conversion)?

Reply:
No, it doesn't pass any navigation to Geant. It just crashes.